main/resource-access.bs

```diff
@@ -75,6 +75,352 @@ Issue: Write Linked Data Platform section.
+
+An auxiliary resource may provide supplementary information about a given
+Solid resource, or affect how that resource and others associated with it
+are processed, served, or interpreted. Different auxiliary resource types
+provide different capabilities. This section introduces a mechanism for linking
+auxiliary resources with regular Solid resources.
+
+Auxiliary resources are needed to influence the configuration, processing, or
+interpretation of Solid resources without changing the composition of the
+resources themselves. To do so would be undesirable in many cases, and not
+possible in others. Auxiliary resources are not meant to replace the ability
+of a Solid resource to self-describe.
+
+Examples of auxiliary resources in use include:
+
+<ul>
+  <li>A binary JPEG image linked to an auxiliary resource that includes
+  information describing that binary JPEG.</li>
+  <li>A container linked to an auxiliary resource that includes access control
+  statements for that container and the resources that belong to it.</li>
+  <li>A resource representation whose shape is constrained by a given ShEx
+  schema that links to an auxiliary resource defining that schema.</li>
+  <li>A resource with an associated set of configurable parameters links to an
+  auxiliary resource where those configurable parameters reside.</li>
+</ul>
+
+### Required Server-side Implementation ### {#ar-server}
+
+
+For any defined auxiliary resource available for a given Solid resource, all
+representations of that resource MUST include an HTTP `Link` header pointing to
+the location of each auxiliary resource.
+
+The `rel={relation-type}` [[!RFC8288]] will define the relationship to the
+target URL in the HTTP `Link` header. URIs are encouraged to indicate Link
+relation types.
+
+An auxiliary resource linked with a given Solid resource through an HTTP `Link`
+header is considered to be *directly associated* with that resource. It is up
+to the server to determine how that association affects processing based on the
+auxiliary resource type.
+
+A given Solid resource MAY link to zero or more auxiliary resources. A given
+Solid resource MAY Link to auxiliary resources on a different server under a
+different authority, per the configuration of the Solid server on which that
+resource resides.
+
+Access to different types of auxiliary resources require varying levels of
+authorization, which MUST be specified as part of the definition for a given
+auxiliary resource type.
+
+An auxiliary resource that resides on a Solid server MUST adhere to the same
+interaction model used by other regular Solid resources, except where specified
+in the definition of that auxiliary resource type.
+
+### Required Client-side Implementation ### {#ar-client}
+
+#### Discovery of Auxiliary Resources #### {#ar-discovery}
+
+To discover the auxiliary resources directly associated with a given Solid
+resource, a Solid client MUST issue a `HEAD` or `GET` request to the target
```

Comment:
How do we feel about an OPTIONS request as well?

Reply:
https://www.mnot.net/blog/2012/10/29/NO_OPTIONS

Reply:
Ahhh got it, thank you. That's very helpful.